homeassistant/components/unifiprotect/views.py

```diff
@@ -226,3 +235,56 @@ async def iterator(total: int, chunk: bytes | None) -> None:
         if response.prepared:
             await response.write_eof()
         return response
+
+
+class VideoEventProxyView(ProtectProxyView):
+    """View to proxy video clips for events from UniFi Protect."""
+
+    url = "/api/unifiprotect/video/{nvr_id}/{event_id}"
```

Comment:
I'm not sure if I understood that correctly, but I get a 404 using this url with an nvr & event id

Reply:
ack - unfortunately forget to stage and add the _init_.py file of unify protect. PTAL now with the latest commit